client/components/banner/style.scss

```diff
@@ -8,37 +8,37 @@
 		padding-right: 48px;
 	}
 
-	@include banner-color( $blue-wordpress );
+	@include banner-color( var( --color-primary ) );
 
 	&.is-wpcom-plan {
 		&.is-upgrade-blogger {
-			@include banner-color(darken($alert-green, 20%));
+			@include banner-color( var( --color-success-dark ) );
```

Comment:
If possible, we may should try to go with the original intent — I think they were trying to simulate the new Celadon color.
```suggestion
			@include banner-color( $muriel-celadon-500 );
```
This might be a case to include a new semantic var, maybe something like `--color-plan-blogger`

![screen shot 2018-12-17 at 4 27 07 pm](https://user-images.githubusercontent.com/618551/50119579-cb9c4e80-0218-11e9-974b-5650d022187d.png)